### PR TITLE
Enable/Disable 'Refresh SVN status cache' job

### PR DESCRIPTION
**Pull request by selsemore**
Add a new 'Ignore refresh SVN status cache' preference. If true, the
'Refresh SVN status cache' job does not get executed when workspace file
changes are made.

This is a patch from the changes.zip file that marcgise provided in the
issue. I've just rebased it and also moved the 'ignore refresh' check
down to just before the status cache refresh job would be scheduled.
That way the auto sharing of SVN projects will still occur.

Issue #48

### CODE REVIEW

**Comment by markphip** [comment]
What will the workflow be when this preference is turned on. If I just edit a file and save I assume it will not be flagged as modified?  What is the process to make the changes be recognized?

Could this preference eventually be something you could set for a project and store in the .settings folder?  So that maybe you turn this on for a project where it is needed but not for all of them?

File: bundles/subclipse.ui/src/org/tigris/subversion/subclipse/ui/messages.properties [hunk 1]
```diff
@@ -1147,6 +1147,7 @@ SVNPreferencePage.javasvn=SVNKit (Pure Java)
 SVNPreferencePage.timeoutValue=&Communication timeout (in seconds):
 SVNPreferencesPage.0=Ignore changes to hidden resources
 SVNPreferencesPage.1=Ignore managed derived resources
+SVNPreferencesPage.2=Ignore refresh SVN status cache
```
Comment (markphip) [inline bundles/subclipse.ui/src/org/tigris/subversion/subclipse/ui/messages.properties]: Maybe it should be "Disable auto refresh of SVN status cache"

**Comment by markphip** [comment]
Other than maybe changing the preference text it looks good to me.

**Comment by selsemore** [comment]
> What will the workflow be when this preference is turned on. If I just edit a file and save I assume it will not be flagged as modified? What is the process to make the changes be recognized?
> 
> Could this preference eventually be something you could set for a project and store in the .settings folder? So that maybe you turn this on for a project where it is needed but not for all of them?

I guess the workflow would be to turn on this preference prior to doing whatever operation it is that doesn't deal with it well, and then turn it right back off after that operation completes.

True, if this preference is on, then editing and saving a file will not flag it as modified.  You would need to use the Team->Refresh/Cleanup option and Refresh.

Being able to set it at the project level is an interesting idea, assuming the problem isn't related to importing the project to begin with.

**Pull request merged**